cfn-resources/stream-instance/test/stream-instance.sample-cfn-request.json

```diff
@@ -0,0 +1 @@
+
```

Comment:
was left empty, leaving an example here can be useful to be able to run requests against a local lambda.

Reply:
Fixed! THanks for raising this